clang/test/SemaSYCL/unnamed-kernel.cpp

```diff
@@ -0,0 +1,68 @@
+// RUN: %clang -I %S/Inputs -fsycl-device-only -Xclang -fsycl-int-header=%t.h -fsyntax-only %s -Xclang -verify %s
```

Comment:
Please, call front-end directly:
```suggestion
// RUN: %clang_cc1 ... %s
```

Reply:
Thanks!

I've also realized that because of presence both `-fsyntax-only` and `-verify` the latter one wasn't actually effective. Now this test really checks something